components/Nav.vue

```diff
@@ -12,6 +12,7 @@
           >
             {{ $t(link.display) }}
           </nuxt-link>
+          <LanguageSelector />
```

Comment:
Mobile shows repeated language selector. need logic to check if not `showMobileLinks`?
![image](https://user-images.githubusercontent.com/23413788/76645557-767fe300-6516-11ea-8e75-2399272ffb7f.png)